### ISSUE TITLE
Integrate something like friendly-id to make nice, readable URL's.

### ISSUE THREAD

**Issue by sergiotapia** [PCMRating/pcmrating#43]
```
http://pcmrating.com/games/show/245620
http://pcmrating.com/batman-arkham-knight-pc-reviews
```

We can use something like Friendly-ID to make the urls nice to read and incidentally increase our position in search engines. Sound good @Lockyy ?

https://github.com/norman/friendly_id

**Comment by sergiotapia** [comment]
Fixed in #47 - no more ugly url's. Bring that google juice!

![](http://gregladen.com/blog/wp-content/uploads/2015/03/Google-Juice.jpg)

**Comment by tjhorner** [comment]
> Bring that google juice!

lol
